packages/bundler/src/es6-rewriter.ts

```diff
@@ -188,18 +202,50 @@ export class Es6Rewriter {
     return resolutions;
   }
 
+  rewriteEs6SourceUrlsToResolved(
+      node: babel.Node, jsImportResolvedUrls: Map<string, ResolvedUrl>) {
+    const bundler = this.bundler;
+    const rewriteDeclarationSource = {
+      enter(
+          path:
+              NodePath<babel.ExportNamedDeclaration|babel.ImportDeclaration>) {
```

Comment:
Does this also need to handle ExportAllDeclarations?

Reply:
Ugh. Rollup chokes on those:
```
export { b } from 'file:///memory/b.js';



var d = ({
        *file:///memory/b.js: *
});

export { d as $d };
```

Working on finding a version of Rollup that does the right thing.  Different versions choking in different ways... :(

Reply:
Looks like this was resolved.